src/lib/components/Assessment/_styles.scss

```diff
@@ -0,0 +1,119 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/tools/breakpoints';
+
+.web-assessment__launcher {
+  display: flex;
+  flex-direction: column;
+  margin: 3em 0;
+}
+
+// Allows buttons to resize with root font-size, maximizing space for questions
+.web-assessment__button {
+  height: 3rem;
+}
+
+web-assessment {
+  display: none;
+  flex-direction: column;
+
+  &[animatable] {
+    animation: .25s ease reverse modal-slideup;
+  }
+
+  &[animatable][open] {
+    animation: .25s ease modal-slideup;
+  }
+
+  // Prevent FOUC by hiding children until after firstUpdated().
+  // This isn't ":not(:defined)" because firstUpdated occurs after a frame.
+  &.unresolved * {
+    display: none !important;
+  }
+
+  @include bp(sm) {
+    & {
+      display: flex;
+      margin: 5em 0 !important; // Override .w-callout
+      padding: 0 !important; // Override .w-callout
+    }
+  }
+}
+
+@keyframes modal-slideup {
+  0% {
+    transform: translatey(40%);
+    opacity: 0;
+  }
+
+  100% {
+    transform: none;
+    opacity: 1;
+  }
+}
+
+// Overrides .w-button--primary so button matches self-assessment color
+.web-assessment__open,
+.web-assessment__open:hover,
+.web-assessment__open:active,
+.web-assessment__open:focus {
+  background: $SELF_ASSESSMENT_PRIMARY_COLOR;
```

Comment:
Do we need all these extra states? I imagine the background wins?

Reply:
Yes, we need these to override `.w-button--primary`.

Reply:
Made a note.